Makefile

```diff
@@ -18,7 +18,7 @@ group=$(shell id -g -n)
 
 export GOBIN=$(PWD)/bin
 
-LD_FLAGS="-w -X main.version=$(VERSION)"
+LD_FLAGS="-w -X main.version=$(VERSION) -extldflags \"-static\""
```

Comment:
how do y'all feel about always statically compiling C code into Go binaries? i added this since on distroless we don't have some C libraries that Go wants to dynamically load (i think they are related to somewhere around `storage/sql/sqlite.go`)

this looks like it would add XXX to the default alpine-based Dex container image

Reply:
Looks good to me